tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/assign/AssignStmtTest.java

```diff
@@ -209,6 +209,20 @@ public void testAssignmentNegativeCases() {
         BAssertUtil.validateError(resultNegative, i++,
                 "incompatible types: expected 'function ((any|error)...) returns ()', found " +
                         "'function (any...) returns ()'", 123, 47);
+        BAssertUtil.validateError(resultNegative, i++,
```

Comment:
I don't think this is the correct file for these tests. Shall we add them in org.ballerinalang.test.expressions.elvis.ElvisExpressionTest?

Reply:
Yes. I will add.

Reply:
Added. b70cbfef6ca3cfccbad57d8164f0ad5a4b62b134